packages/python-google-compute-engine/google_compute_engine/distro_lib/helpers.py

```diff
@@ -98,3 +121,18 @@ def CallNtpdate(logger):
     logger.warning('Failed to sync system time with ntp server.')
   else:
     logger.info('Synced system time with ntp server.')
+
+def CallWriteViaSysCtl(logger, name, value):
+  """Write a variable using sysctl,
+  Args:
+      name: name of the sysctl variable.
+      value: value of the sysctl variable.
+  """
+  logger.info('Configuring sysctl %s.', name)
+
+  sysctl_command = ['sysctl', '-w',
```

Comment:
Please line wrap after the open bracket. Alternatively, you can replace this with:
```sysctl_command = ['sysctl', '-w', '%s=%s' % (name, value)]```